### DISCUSSION TITLE
How to understand the magnE values in a tDCS simulation?

### DISCUSSION THREAD

**Discussion by Raichuu41** [simnibs/simnibs#234] [9. Results and analysis] [answered]
I have a tDCS simulation which has very small currents of 1 mA in some electrodes. Based on the documentation, the magnE value is stored in V/m. I want to do a line integral from one start point to an end point on the mesh and at the end I would expect a small voltage difference in the range of 1-10 volts at max. However, I get results of over 983 volt.
I tried many different approaches to calculate a line integral for the voltage difference but all of them have been exceedingly high. So I checked the magnE values in particular and they are really high, to give a sample of magnE values from the intersecting elements in the line:
```
    000 = {float64: ()} 373575.92097860086
    001 = {float64: ()} 243985.57668214696
    002 = {float64: ()} 854361.7345256626
    003 = {float64: ()} 854361.7345256626
    004 = {float64: ()} 136957.0017405795
```
Is there some additional step required to "understand" the magnE values, something that SimNIBS does before it displays it in the post-processing view of gmsh? 
Here's some of my sample code of how I retrieve the values:
```python
# main logical flow
gmsh.intialize()
gmsh.open(<path to _scalar.msh file>)
view_tags = get_view_tags()  # custom function to return the view tags in the format {"name": <tag number>}
extract_data(tag=view_tags["magnE"])
```

```python
# relevant parts of extract data implementation
def extract_data(tag):
    # using magnE tag here
    model_data = gmsh.view.getModelData(tag, 0)
    element_tags = model_data[1]
    magn_e_values = model_data[2]  # <-- list of numpy (1,) arrays
    magn_e_values = [val[0] for val in magn_e_values]  # <-- get the values into one list together
``` 

magn_e_values is a flat numpy array - but why are the values extremely high? How to read/understand these values?

**Comment by TorgeW** [comment, marked as answer]
see https://github.com/simnibs/simnibs/discussions/230 for now.
